ecosystem/readme.md

```diff
@@ -1,14 +1,118 @@
+# Stellar Ecosystem Proposals (SEPs)
 
-Table of Contents:
-- [sep-0001](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0001.md) stellar.toml file
-- [sep-0002](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0002.md) Federation protocol
-- [sep-0003](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0003.md) Compliance protocol
-- [sep-0004](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0004.md) Tx Status Endpoint
-- [sep-0005](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0005.md) Key Derivation Methods for Stellar Accounts
-- [sep-0006](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md) Anchor/Client interoperability
-- [sep-0007](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0007.md) URI Scheme to facilitate delegated signing
-- [sep-0008](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0008.md) Regulated Assets
-- [sep-0009](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0009.md) Standard KYC / AML fields
-- [sep-0010](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md) Stellar Web Authentication
-- [sep-0011](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0011.md) Txrep: human-readable low-level representation of Stellar transactions
-- [sep-0012](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md) Anchor/Client customer info transfer
+## SEP Track Terms
+* **Informational** — A SEP on the `Informational` track is one that is open to adoption by the
+  ecosystem, but has not been formally standardized by SDF, and is not endorsed by SDF for
+  adoption. Typically a SEP can start as `Informational` to gain traction within the ecosystem
+  before moving to the `Standards` track.
+* **Standard** — A SEP on the `Standards` track is one that aims for formal standardization and
+  endorsement by SDF for adoption. Typically a Standard has a higher bar towards acceptance, and it
+  requires approval by 2 SDF members of the SEP Team.
+
+## SEP Status Terms
+* **Archived** - A SEP that did not head towards a disposition due to a lack of consensus _and_
+  support. Generally open to revival with additional edits.
+* **Draft** - A SEP that is currently open to discussion and feedback.
+* **FCP** - A SEP that has entered the Final Comment Period (FCP), and has one week remaining
```

Comment:
My experiences implementing SEP-6 says that there needs to be more feedback after a final draft has been reached. We made several significant changes to the spec (data shape, /fee endpoint) due to issues discovered during implementation.

It might be worth gating "ready for wide adoption" behind a limited number of prototype implementations to try to discover needs that weren't anticipated. A hands-on implementation with a key partner and members of the SEP team seems like it would give a lot of opportunity to optimize and clarify the spec text.

Reply:
@tomquisel as well.

One thing I did was add an "Active" category where a SEP can continue to be updated, but is generally worth putting out into the world. The TOML file for example would always be "Active". Things should naturally graduate towards "Final" when they can, and I think honestly we can beta test Draft proposals out? Happy for additional feedback though, I do like this idea.